extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -913,20 +912,19 @@ export class SchemaCompareMainWindow {
 					this.applyButton.enabled = true;
 					this.applyButton.title = loc.applyEnabledMessage;
 				}
+				else if (this.targetEndpointInfo.endpointType === mssql.SchemaCompareEndpointType.Project) {
```

Comment:
nit: else if should be on the same line as the } on the line above

Reply:
whoops; been working in DacFx for too long